ScreenToGif/Resources/Localization/StringResources.de.xaml

```diff
@@ -222,13 +228,13 @@
     <s:String x:Key="S.Options.App.Tray.Interactions.Else">ansonsten:</s:String>
     <s:String x:Key="S.Options.App.General">Allgemein</s:String>
     <s:String x:Key="S.Options.App.General.WorkaroundQuota">Behelfslösung für den Absturz von „Nicht genügend Speicherplatz …”</s:String>
-    <s:String x:Key="S.Options.App.General.WorkaroundQuota.Info">(Experimentell)</s:String>
+    <s:String x:Key="S.Options.App.General.WorkaroundQuota.Info">(experimentell)</s:String>

```

Comment:
The title case in here is because it's the first letter, like in other .Info below.